packages/sdk/src/handlers/contract-kit.ts

```diff
@@ -102,6 +102,7 @@ export class ContractKitTransactionHandler implements ChainHandler {
     txo: CeloTxObject<unknown>
   ) {
```

Comment:
Would be good if this was Typed explicitly as I think `CeloTx` from celo/connect/types